pkg/controllers/leaderworkerset_controller.go

```diff
@@ -403,7 +408,7 @@ func (r *LeaderWorkerSetReconciler) updateStatus(ctx context.Context, lws *leade
 	sts := &appsv1.StatefulSet{}
 	if err := r.Get(ctx, types.NamespacedName{Name: lws.Name, Namespace: lws.Namespace}, sts); err != nil {
 		log.Error(err, "Error retrieving leader StatefulSet")
-		return err
+		return client.IgnoreNotFound(err)
```

Comment:
I think we need to throw the error here if the leader statefulset exist, since it's supposed to be created already at this point?

Reply:
done